bookshelf/app.yaml

```diff
@@ -0,0 +1,12 @@
+runtime: custom
+vm: true
+
+manual_scaling:
```

Comment:
I think `scaling` topic is important and worthwhile to explain in the guide. If we have such a section anyways, it's not an issue.